app_dart/lib/src/model/appengine/commit.dart

```diff
@@ -50,6 +51,10 @@ class Commit extends Model {
   @StringProperty(propertyName: 'FlutterRepositoryPath', required: true)
   String repository;
 
+  /// The branch of the commit.
+  @StringProperty(propertyName: 'Branch', required: true)
```

Comment:
How does making this required play with the existing entries that don't have it?

Reply:
For the safety purpose, removing `required`. Thanks.